Assets/Scripts/Items/Caffeine.cs

```diff
@@ -1,14 +1,18 @@
-﻿using UnityEngine;
+﻿using Combat;
+using UnityEngine;
 
 namespace Items {
     [DisallowMultipleComponent]
     [RequireComponent(typeof(Interactable))]
     public class Caffeine : MonoBehaviour
     {
+        [SerializeField] public int enhancedValue = 1;
```

Comment:
This attribute is not needed here as it is already automatically serialized.

```suggestion
        public int enhancedValue = 1;
```